ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+# Backup collections
```

Comment:
Add "backup collections" entry to glossary.md with a brief description and a link to this article for further details.